projects/step-frontend/src/lib/modules/scheduler/execution-task-parameters-list/execution-task-parameters-list.component.ts

```diff
@@ -0,0 +1,141 @@
+import { Location } from '@angular/common';
+import { DashboardService } from '../../_common/services/dashboard.service';
+
+type InProgress = Mutable<Pick<ExecutionTaskParametersListComponent, 'inProgress'>>;
+
+@Component({
+  selector: 'step-execution-task-parameters-list',
+  templateUrl: './execution-task-parameters-list.component.html',
+  styleUrls: ['./execution-task-parameters-list.component.scss'],
+})
+export class ExecutionTaskParametersListComponent {
+  readonly STATUS_ACTIVE_STRING = 'On';
+  readonly STATUS_INACTIVE_STRING = 'Off';
+
+  readonly currentUserName: string;
+  readonly inProgress: boolean = false;
+
+  private executionTaskParametersRequest$ = new BehaviorSubject<any>({});
+  private executionTaskParameters$ = this.executionTaskParametersRequest$.pipe(
+    tap((_) => ((this as InProgress).inProgress = true)),
+    switchMap((_) => this._schedulerService.getScheduledExecutions()),
+    tap((_) => ((this as InProgress).inProgress = false)),
+    shareReplay(1)
+  );
+
+  readonly searchableExecutionTaskParameters$ = new TableLocalDataSource(this.executionTaskParameters$, {
+    searchPredicates: {
+      name: (element, searchValue) => element.attributes!['name'].toLowerCase().includes(searchValue.toLowerCase()),
+      environment: (element, searchValue) =>
+        element.executionsParameters!.customParameters!['env'].toLowerCase().includes(searchValue.toLowerCase()),
+      cronExpression: (element, searchValue) =>
+        element.cronExpression!.toLowerCase().includes(searchValue.toLowerCase()),
+      status: (element, searchValue) =>
+        element.active
+          ? this.STATUS_ACTIVE_STRING.toLowerCase().includes(searchValue.toLowerCase())
+          : this.STATUS_INACTIVE_STRING.toLowerCase().includes(searchValue.toLowerCase()),
+    },
+    sortPredicates: {
+      name: (elementA, elementB) => elementA.attributes!['name'].localeCompare(elementB.attributes!['name']),
+      environment: (elementA, elementB) =>
+        elementA.executionsParameters!.customParameters!['env'].localeCompare(
+          elementB.executionsParameters!.customParameters!['env']
+        ),
+      status: (elementA, elementB) => +elementB.active - +elementA.active,
+    },
+  });
+
+  constructor(
+    private _dashboardService: DashboardService,
+    private _schedulerService: SchedulerService,
+    private _dialogs: DialogsService,
+    private _executionTaskParameterDialogs: ExecutionTaskParameterDialogsService,
+    private _isUsedByDialogs: IsUsedByDialogsService,
+    private _auth: AuthService,
+    private _tableRest: TableRestService,
+    public _location: Location,
+    private _httpClient: HttpClient,
+    context: ContextService
+  ) {
+    this.currentUserName = context.userName;
+  }
+
+  private loadTable(): void {
+    this.executionTaskParametersRequest$.next({});
+  }
+
+  executeParameter(executionTaskParameters: ExecutiontTaskParameters) {
+    //@ts-ignore
```

Comment:
The method of the generated `_schedulerService` could not be used in this case, becasue the API is returning a plain string instead of a JSON. That should be changed on API side.
Anyways, the way I worked around that is not ideal. In my opinion, the ideal solution would be to create an `AugmentedSchedulerService` like I've been showing in #41  , since this is code that belongs to the API-client. If eventually the API would return a JSON, we would just need to remove the augmentation.